src/stream.rs

```diff
@@ -18,8 +18,8 @@ pub enum StreamConnector {
     /// Connect to the server through a TCP stream encrypted with SSL.
     Ssl {
         ca_file: String,
-        certificate_file: String,
-        key_file: String,
+        certificate_file: Option<String>,
```

Comment:
Given that this is public, I think it's worth adding another paragraph in the documentation for StreamConnector::Ssl noting that it's not valid to have one of the options populated but not the other.

Reply:
Done.